source/features/conventional-commits.css

```diff
@@ -0,0 +1,85 @@
+.rgh-commit-type-label-feat-colors {
+	--label-r: 14;
+	--label-g: 138;
+	--label-b: 22;
+	--label-h: 123;
+	--label-s: 81;
+	--label-l: 29;
+}
+
+.rgh-commit-type-label-fix-colors {
+	--label-r: 215;
+	--label-g: 58;
+	--label-b: 74;
+	--label-h: 353;
+	--label-s: 66;
+	--label-l: 53;
+}
+
+.rgh-commit-type-label-chore-colors {
+	--label-r: 100;
+	--label-g: 100;
+	--label-b: 100;
+	--label-h: 0;
+	--label-s: 0;
+	--label-l: 255;
+}
+
+.rgh-commit-type-label-docs-colors {
+	--label-r: 23;
+	--label-g: 157;
+	--label-b: 125;
+	--label-h: 165;
+	--label-s: 74;
+	--label-l: 35;
+}
+
+.rgh-commit-type-label-build-colors {
+	--label-r: 255;
+	--label-g: 150;
+	--label-b: 0;
+	--label-h: 36;
+	--label-s: 100;
+	--label-l: 50;
+}
+
+.rgh-commit-type-label-refactor-colors {
+	--label-r: 165;
+	--label-g: 138;
+	--label-b: 82;
+	--label-h: 40;
+	--label-s: 33;
+	--label-l: 48;
```

Comment:
Can you pick a better color for this? Black on mustard isn’t very good

Reply:
I went with placeholder colors. Someone dedicated may know better and more standard color combinations for GitHub.

Reply:
![image](https://github.com/user-attachments/assets/ec1dc39d-eb51-49c5-a95a-932fde87b22c)

how about this color

Reply:
The previous color looked good in dark mode but not in light mode

Reply:
![image](https://github.com/user-attachments/assets/cb919e78-333f-49c2-b100-45ea0cf9a4e9)

I've updated the colors to the default palette GitHub suggests

Reply:
One thing to consider is to switch the ci and chore colors. chore seems to be used more often but has a more annoying color.

Reply:
Which is why I’d rather see it in neutral/gray color. Anyway this is good enough for now, this week I’ll be tweaking the feature after merge.